aws/network.ts

```diff
@@ -128,24 +128,31 @@ export class Network extends pulumi.ComponentResource {
      * default network will be lazily created, using whatever options are provided in opts. All
      * subsequent calls will return that same network even if different opts are provided.
      */
-    public static getDefault(opts?: pulumi.ComponentResourceOptions): Network {
+    public static getDefault(opts?: pulumi.ComponentResourceOptions): Promise<Network> {
```

Comment:
Minor - but could probably make this `async/await` to clean up a bit of the promise code in here.